### PR TITLE
Increase test coverage on Jib Builder

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

**Comment by dgageot** [comment]
ping @TadCordle @briandealwis @loosebazooka @chanseokoh

File: testutil/util.go [hunk 1]
```diff
@@ -35,6 +35,10 @@ type T struct {
 	teardownActions []func()
 }
 
+func (t *T) FakeRun(command string) *FakeCmd {
```
Comment (TadCordle) [inline testutil/util.go]: Is this used anywhere?

Comment (dgageot) [reply testutil/util.go]: oops, should have been. Let me push an update

File: pkg/skaffold/build/local/jib_gradle_test.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		description   string
+		artifact      *latest.JibGradleArtifact
+		cmd           util.Command
+		shouldErr     bool
+		expectedError string
+	}{
+		{
+			description: "local build",
+			artifact:    &latest.JibGradleArtifact{},
+			cmd:         testutil.FakeRun(t, "gradle -Djib.console=plain :jibDockerBuild --image=img:tag"),
```
Comment (loosebazooka) [inline pkg/skaffold/build/local/jib_gradle_test.go]: I'm not sure how this testing framework works: Are we verifying that the code creates the command defined in FakeRun `"gradle -Djib.console=plain :jibDockerBuild --image=img:tag"`?

**Comment by dgageot** [comment]
@TadCordle should be good now. Thanks for the review!

File: pkg/skaffold/build/local/jib_gradle_test.go (hunk 2 above)
Comment (dgageot) [reply pkg/skaffold/build/local/jib_gradle_test.go]: It’s checking that command and only this one runs

Comment (loosebazooka) [reply pkg/skaffold/build/local/jib_gradle_test.go]: kinda like a `Mockito.when().then()`?

Comment (dgageot) [reply pkg/skaffold/build/local/jib_gradle_test.go]: Yes.

**Comment by dgageot** [comment]
Don’t worry, I’m working on the remote case but I’d rather get this merged first

**Comment by loosebazooka** [comment]
Is jib team not owners?

**Comment by dgageot** [comment]
That would make sense but the file pattern `**/**jib**` which doesn't to match `jib_maven_test.go`...

**Pull request merged**